satellite/satellitedb/accounting.go

```diff
@@ -162,6 +162,28 @@ func (db *accountingDB) SaveRollup(ctx context.Context, latestRollup time.Time,
 	return Error.Wrap(err)
 }
 
+// SaveBucketInfo saves the latest bucket info
+func (db *accountingDB) SaveBucketInfo(ctx context.Context, intervalStart time.Time, bucketInfo map[string]*accounting.BucketStats) error {
+	if len(bucketInfo) == 0 {
+		return Error.New("In SaveBucketInfo with empty bucketInfo")
+	}
+	for k, v := range bucketInfo {
```

Comment:
A couple thoughts here:
1. the name of this function gives the impression it is only saving the info for one bucket, but it is actually saving the info for all the buckets (or at least a list of buckets).  Should we rename the function something like `SaveAllBucketsInfo` or some name to indicate it is more than one bucket?

2. I feel like `accounting.BucketStats` should actually be called `accounting.BucketTally` and should mirror the `dbx.BucketStorageTally` struct.  Then the argument to this function would be something like:
`bucketsTallyInfo map[string]*accounting.BucketTally` 

3. can we rename `k, v` to be more descriptive like `bucketID, bucketTallyInfo`